django_colortag/locale/fi/LC_MESSAGES/django.po

```diff
@@ -17,28 +17,81 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "X-Generator: Lokalize 2.0\n"
 
-#: django_colortag_/models.py:23
+#: django_colortag/models.py
 msgid "Display name for tag"
 msgstr "Tagin näyttönimi"
 
-#: django_colortag_/models.py:24
+#: django_colortag/models.py
 msgid "Slug key for tag. If left blank, one is created from name"
 msgstr ""
 "Yksisanainen tunniste tagille. Jos jätät tyhjäksi, sopiva luodaan tagin "
 "nimestä."
 
-#: django_colortag_/models.py:25
+#: django_colortag/models.py
 msgid "Describe the usage or meaning of this tag"
 msgstr "Kuvaile tagin käyttö tai merkitys."
 
-#: django_colortag_/models.py:26
+#: django_colortag/models.py
 msgid "Color that is used as background for this tag"
 msgstr "Tagin taustaväri"
 
-#: django_colortag_/widgets.py:162
-msgid "Match any (OR)"
-msgstr "Sisällä osa (TAI)"
+#: django_colortag/templates/django_colortag/widgets/colortag_andor.html
+msgid "Show instructions"
+msgstr "Näytä ohjeet"
 
-#: django_colortag_/widgets.py:163
-msgid "Match all (AND)"
-msgstr "Sisällä kaikki (JA)"
+#: django_colortag/widgets.py
+msgid "Show a result if it has ANY of the selected tags."
+msgstr "Näytä hakutulos, jos sillä on JOKIN valituista tageista."
+
+#: django_colortag/widgets.py
+msgid "Show a result only if it has ALL of the selected tags."
+msgstr "Näytä hakutulos vain, jos sillä on KAIKKI valituista tageista."
+
+#: django_colortag/widgets.py
+msgid "Colortag filter"
+msgstr "Väritagisuodatin"
```

Comment:
In MOOC-Jutut `tag` in Finnish has been written as `täg` but here it is `tag`:
https://github.com/search?q=repo%3Aapluslms%2Fmooc-jutut%20t%C3%A4g&type=code

Which one should we be using here?

Reply:
That's a very good question and I was wondering the same. I noticed that in the earlier translations of this repo, 'tagi' had been used, which was why I used 'tagi' instead of 'tägi'.
Also, in Kielitoimiston sanakirja, [tagi](https://www.kielitoimistonsanakirja.fi/#/tagi) is more extensively defined, while there the definition of ['tägi'](https://www.kielitoimistonsanakirja.fi/#/t%C3%A4gi) just links to 'tagi'. So I suppose someone might ask if it would be better to also start using "tagi" in MOOC-Jutut.

Reply:
And meanwhile in Aplus we refer to them as "merkintä". So we could also ponder if that should be the word used instead in both here and MOOC-Jutut.

Reply:
I'm down to use "tagi" everywhere.